test_all_commands.sh

```
@@ -61,7 +61,31 @@ uv run simba preprocess \
 echo ""
-echo "3/10 Testing: simba train"
+echo "3/12 Testing: simba-sweep (fresh, 3 trials, 1 epoch)"
+echo "------------------------------------------------------------"
+uv run simba-sweep \
+    +sweep=default \
+    sweep.n_trials=3 \
+    sweep.output_dir=./test_full_workflow/sweep_run1 \
+    sweep.resume=false \
+    paths.preprocessing_dir_train=./test_full_workflow/preprocessed/ \
+    training.epochs=1 \
+    hardware.accelerator=$DEVICE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)